plugins/CoreHome/vue/src/CopyToClipboard/CopyToClipboard.ts

```diff
@@ -26,6 +26,7 @@ function onClickHandler(pre: HTMLElement) {
     textarea.style.left = '-9999px';
     document.body.appendChild(textarea);
     textarea.select();
+    textarea.focus();
```

Comment:
Would it make more sense to focus first and then select? Just purely from the perspective of what happens when a user selects text in a textarea.

Reply:
@michalkleiner No preference over it as it needs focus just before `document.execCommand('copy');` is executed for it to work always, so added there and won't make any harm I believe even if change to what you are suggesting.